application/aam-backend-service/src/main/kotlin/com/aamdigital/aambackendservice/skill/skilllab/SkillLabUserProfileMatcher.kt

```diff
@@ -0,0 +1,13 @@
+package com.aamdigital.aambackendservice.skill.skilllab
+
+import com.aamdigital.aambackendservice.skill.core.UserProfileMatcher
+import com.aamdigital.aambackendservice.skill.domain.UserProfile
+import org.springframework.web.client.RestClient
+
+class SkillLabUserProfileMatcher(
+    val http: RestClient
+) : UserProfileMatcher {
+    override fun findExternalUserProfiles(userProfile: UserProfile) {
+        TODO("Not yet implemented")
```

Comment:
tagging open TODO (or can we use a generic implementation, not requiring a SkillLab specific matcher?)